src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  width: 100px;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-around;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    box-sizing: border-box;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+
+    &:nth-child(-n + 4) {
```

Comment:
Change here to use the modifier `stars--N`, you can check the comment in line 44 to 49 to understand better about it.